app/lib/file_parse_service.rb

```diff
@@ -99,27 +99,25 @@ def self.run_parse_job(study_file, study, user, reparse: false, persist_on_fail:
           Rails.logger.info "Aborting parse of #{@study_file.name} as #{@study_file.file_type} in study #{@study.name}; not applicable"
         end
       when 'AnnData'
-        # gate ingest of AnnData using the feature flag 'ingest_anndata_file'
+        # gate full ingest of an AnnData file using the feature flag 'ingest_anndata_file'
         if do_anndata_file_ingest == true
```

Comment:
Redundant - this would evaluate to `if true == true`
```suggestion
        if do_anndata_file_ingest
```